tests/functional/adapter/test_persist_docs.py

```diff
@@ -62,3 +78,30 @@ def test_comment_on_late_binding_view(self, project):
 
         no_docs_node = catalog_data["nodes"]["model.test.no_docs_model"]
         self._assert_has_view_comments(no_docs_node, False, False)
+
+
+class TestPersistDocsWithMaterializedView(BasePersistDocs):
+    @pytest.fixture(scope="class", autouse=True)
+    def seeds(self):
+        return {"my_seed.csv": files.MY_SEED}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "my_materialized_view.sql": files.MY_MATERIALIZED_VIEW,
+        }
+
+    @pytest.fixture(scope="class")
+    def properties(self):
+        return {
+            "schema.yml": _MATERIALIZED_VIEW_PROPERTIES__SCHEMA_YML,
+        }
+
+    def test_has_comments_pglike(self, project):
```

Comment:
@jeremyyeo Please add the `@pytest.mark.flaky` decorator to this test case. We have found that these tests are flaky when run in parallel and this decorator reroutes this test into a serialized test suite.